sdk/src/env.rs

```diff
@@ -77,12 +76,13 @@ impl Env {
         }
     }
 
-    #[cfg(feature = "testutils")]
-    pub fn push_test_frame(&self, contract_id: ArrayBinary<32>) -> FrameGuard {
-        self.env_impl
-            .push_test_frame(contract_id.into_val(self))
-            .unwrap()
-    }
+    // TODO: push_test_frame is no longer a pub method, and this needs updating.
```

Comment:
@jonjove After updating the stellar-contract-env-* dependencies this fn caused a compile error in the SDK. I figure you've already got a fix on its way so I've commented it with a TODO temporarily. You'll probably see this as a conflict in your fix, but hopefully it is not too difficult to resolve.

Reply:
This function no longer exists in my version, so feel free to just delete it.

Reply:
Done!